Source/WebCore/page/Navigation.cpp

```diff
@@ -76,7 +75,7 @@ bool Navigation::canGoForward() const
 void Navigation::initializeEntries(const Ref<HistoryItem>& currentItem, Vector<Ref<HistoryItem>>& items)
 {
     for (Ref item : items)
-        m_entries.append(NavigationHistoryEntry::create(protectedScriptExecutionContext().get(), item));
+        m_entries.append(NavigationHistoryEntry::create(scriptExecutionContext(), item));
```

Comment:
This is a regression, we should be passing a smart pointer here, please call `protectedScriptExecutionContext().get()`. If `protectedScriptExecutionContext()` is missing, please add it.